tests/test_engines.py

```diff
@@ -43,21 +43,6 @@ def modified_boards() -> List[tuple]:
     ]
 
 
-@pytest.fixture
-def minor_boards() -> List[tuple]:
-    """
-    Sets up boards specifically for the tests on PrioritizeKnightMoves
-    and PrioritizeBishopMoves. Includes recommended moves
-
-    Returns:
-        {List[tuple]}
-    """
-    return [
-        (chess.Board(fen=load_fen("bishop_or_knight_capture_queen")), "c1g5"),
-        (chess.Board(fen=load_fen("bishop_or_knight_capture_queen")), "f3g5"),
-    ]
-
-
 @pytest.fixture
 def versatile_board() -> List[tuple]:
```

Comment:
Since this only applies to a single test, I think we can move this into the test itself instead of storing it as a fixture. This would improve readability of the test since everything's close together